config/jobs/cert-manager/cert-manager/cert-manager-periodics.yaml

```diff
@@ -406,10 +371,12 @@ periodics:
       preset-default-e2e-volumes: "true"
       preset-dind-enabled: "true"
       preset-disable-all-alpha-beta-feature-gates: "true"
+      preset-ginkgo-skip-default: "true"
```

Comment:
note: I think the lack of "skip-default" here was unintentional and that this is a fix. We probably don't want to test Venafi stuff in a "feature gates disabled" test.